app/src/behaviors/behavior_soft_off.c

```diff
@@ -57,6 +57,9 @@ static int on_keymap_binding_released(struct zmk_behavior_binding *binding,
         uint32_t hold_time = k_uptime_get() - data->press_start;
 
         if (hold_time > config->hold_time_ms) {
+            if (IS_ENABLED(CONFIG_ZMK_SPLIT) && IS_ENABLED(CONFIG_ZMK_SPLIT_ROLE_CENTRAL)) {
+                k_usleep(100000);
```

Comment:
If you're simply trying to sleep for 100ms, just use:

```suggestion
                k_sleep(K_MSEC(100));
```

Reply:
done!